models/Movie.model.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const movieSchema = new Schema ({
+
+    title: {
+        type: String,
+        required: true,
+        unique: true,
+    },
+    director: {
+        type: String,
+        unique: true,
```

Comment:
I think this is wrong. Director shouldn't be unique, otherwise two different movies, but from the same director would break.